src/components/views/causes/CausesIndex.tsx

```
@@ -0,0 +1,404 @@
+				)}
+				{isFetchingNextPage && <Loader className='dot-flashing' />}
+				{data?.pages.some(page => page.data.length > 0) ? (
+					<ProjectsWrapper>
+						<ProjectsContainer>
+							{isQF ? (
+								<CauseQFMiddleBanner />
+							) : (
+								<CausesMiddleGivethVaultBanner />
+							)}
+							{data.pages.map((page, pageIndex) => (
+								<>nothing for now TODO</>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)